.pre-commit-config.yaml

```
@@ -13,14 +13,15 @@ repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
     rev: v0.3.5 # must match requirements-tests.txt
     hooks:
-      - id: ruff
-        name: Run ruff on stubs, tests and scripts
-        args: ["--exit-non-zero-on-fix"]
       - id: ruff
         # Run this separately because we don't really want
         # to use --unsafe-fixes for all rules
+        # Should be run first as it can leave unused imports behind
         name: Remove unnecessary `sys.version_info` blocks
         args: ["--exit-non-zero-on-fix", "--select=UP036", "--unsafe-fixes"]
+      - id: ruff
+        name: Run ruff on stubs, tests and scripts
+        args: ["--exit-non-zero-on-fix"]
```

```suggestion
        name: Run ruff on stubs, tests and scripts
        args: ["--exit-non-zero-on-fix"]
      - id: ruff
        # Run this separately because we don't really want
        # to use --unsafe-fixes for all rules
        name: Remove unnecessary `sys.version_info` blocks
        args: ["--exit-non-zero-on-fix", "--select=UP036", "--unsafe-fixes"]
```

Reverting changes in this file, moved to https://github.com/python/typeshed/pull/11737